packages/kbn-ui-framework/dist/ui_framework.css

```diff
@@ -2730,6 +2730,8 @@ main {
     .theme-dark .kuiLocalSearchInput:focus {
       outline: none;
       border-color: #0079a5; }
+  .kuiLocalSearchInput:focus {
```

Comment:
I assume this is why the focus indication on the input look different than all the other elements on the page. Should it be styled to look the same?

![mar-13-2018 17-19-31](https://user-images.githubusercontent.com/404731/37376867-bc03a254-26e3-11e8-924c-aa8057462bef.gif)

Reply:
@snide?

Reply:
It had a double outline on this one specifically which was pretty gnarly. I spot fix really bad cases, but ultimately a lot of these issues need to be fixed through EUIfication. Till then I don't plan on updating the accessibility focus styles (which are a pretty aggressive css selector) since we want all this stuff to go out the door.

Reply:
Is it worth moving this to EUI then? Even if we don't use the React components here, perhaps we can use the classes in Angular? Perhaps that's pretty brittle though, I don't know how stable the underlying CSS is...

FWIW, I'm not proposing blocking this PR on it, it's OK, just a little unfortunate that the appearance isn't consistent.

Reply:
The whole thing is pretty brittle and brute forced, for example... here's what the `.less` side of things looks like for kibana when it comes to focus states.

```
:focus {
  &:not([class^="eui"]),
  {
    @include focus;
  }
}
```

Basically, all elements that need some sort of focus that aren't eui based (which has focus / accessibility as a core consideration) get a brute force attack that does the weird double outline (because who knows what the background color of the element is!). It's the only way that I knew to make 6.0 accessible quickly. Obviously this is a terrible way to do things and it makes the visuals totally inconsistent, because if you _don't_ want that brute force style, or if they already had some sort of focus state, they'll be off. So in practice, I've avoided doing that (except in rare cases), knowing that this stuff will automatically get fixed with EUI adoption.

TL;DR

We can do it, but it's brittle and we should just switch stuff over (not in this PR).